CHANGELOG.md

```diff
@@ -30,6 +30,9 @@
 
 ### New
 
+- Add DynamoDB Scaler ([#2486](https://github.com/kedacore/keda/pull/2486))
+- **General:** Automatically release container image for ARM ([#2263]https://github.com/kedacore/keda/issues/2263))
+- **General:** Automatically run end-to-end tests on ARM ([#2262]https://github.com/kedacore/keda/issues/2262))
```

Comment:
Could you please include just the relevant info? and use the format as it is used for other new scalers (please also include AWS in the name)